google-map-search.html

```diff
@@ -153,14 +153,13 @@
         if (this.types && typeof this.types == 'string') {
           var types = this.types.split(' ');
         }
-
-        if (!this.globalSearch) {
-          var bounds = this.map.getBounds();
-        } else if (this.radius) {
+        if (this.radius) {
           var radius = this.radius;
           var location = this.location ? this.location : this.map.getCenter();
+        }else if (!this.globalSearch) {
```

Comment:
I assume you meant there should be a space between the floral brace and 'else'
added one space